README.rst

```diff
@@ -70,9 +70,113 @@ We appreciate any help!
 BUILD & DEVELOPMENT
 -------------------
 
-Run ``python3 setup.py install``
+1. Installation instructions for **Windows** users:
 
-Note: You need to have bower installed before you may install the python packages.
+   - Clone the **coala-html** repository
+
+     ::
+
+       $ git clone https://github.com/coala/coala-html.git
+
+   - Download and install the latest version of Node.js from `official website <https://nodejs.org/en/download/>`__.
+
+   - Change the directory to **coala-html** and run following command to install dependencies
+
+     ::
+
+       $ python setup.py install
+
+   - Install **coala-bears**
+
+     ::
+
+       $ pip install coala-bears
+
+   - Install **bower** to build static web pages
+
+     ::
+
+       $ npm install -g bower
+
+
+   - Run **coala-html** to display results from **coala-json**
+
+     ::
+
+       $ coala-html --dir ..\Tests
+
+
+2. Installation instructions for **Ubuntu/Linux** users:
+
+   - Clone the **coala-html** repository
+
+     ::
+
+      $ git clone https://github.com/coala/coala-html.git
+
+   - Install latest version of **Node.js**
+
+     a. For **Ubuntu** users:
+
+        * *Configure Node.js PPA*
+
+          ::
+
+           $ sudo apt-get install curl python-software-properties
+           $ curl -sL https://deb.nodesource.com/setup_11.x | sudo bash -
+
+        * *Install Node.js and npm*
+
+          ::
+
+           $ sudo apt-get install nodejs
+
+     b. For installing Node.js via Package manager on other distributions, refer `official guide <https://nodejs.org/en/download/package-manager/>`__.
+
+   - *Check Node.js and npm version*
+
+     You can find more details about current version on node.js `official website <https://nodejs.org/>`__.
+
+     ::
+
+       $ node -v
+
+       $ npm -v
+
+   - Install **bower** to build static web pages
+
+     ::
+
+       $ sudo npm install -g bower
+
+   - Change the directory to **coala-html** and run following command to install dependencies
+
+     ::
+
+       $ python3 setup.py install
+
+   - Install **coala-bears**
+
+     ::
+
+       $ pip3 install coala-bears
+
+   - Allow permissions to **root user** to install node packages
```

Comment:
I followed the steps without changing permissions and it still worked. Can you confirm that this `permission change` is **absolutely** necessary?

Reply:
In this section, I found that 2nd command needs to be modifies to `$ sudo chown -R $USER:$GROUP ~/.config/configstore` and rest of the commands are no longer needed. I executed only this instruction because the folder configstore contains the `github-bower.json` which specifies the packages need to be installed and is restricted to read. So need to provide root permissions only for this directory and else commands are no longer needed.
But this 2nd command is necessary to work with coala-html, else it won't work

Reply:
@frextrite Change the heading also, as it's now more specific why to give permission